code/_helpers/game.dm

```diff
@@ -543,3 +543,11 @@ datum/projectile_data
 		if(M.client)
 			viewing += M.client
 	flick_overlay(I, viewing, duration)
+
+/proc/to_chat(target, message)
+	if(ismech(target))
+		var/mob/living/heavy_vehicle/mech = target
+		for(var/pilot in mech.pilots)
+			pilot << message
```

Comment:
Imho it would be better to move that to a `/mob/living/heavy_vehicle/to_chat(target,message)` to avoid having to check if the target is a mech whenever any message on the serve is displayed.